eng/Versions.props

```diff
@@ -44,6 +44,10 @@
       src\Razor\test\testassets\Directory.Build.props when this changes.
     -->
     <DefaultNetCoreTargetFramework>net9.0</DefaultNetCoreTargetFramework>
+    <!-- this should usually match for LTS builds, and be off-by-one otherwise;
+      example use-case here is OOB packages where Aspire wants to be able to consume
+      new features specific to OOB -->
+    <CurrentLtsTargetFramework>net8.0</CurrentLtsTargetFramework>
```

Comment:
Can you add a bullet point to https://github.com/dotnet/aspnetcore/blob/main/docs/UpdatingMajorVersionAndTFM.md#updating-tfm about updating this?